### ISSUE TITLE
Request: Support for Dino Depot for Propagators

### ISSUE THREAD

**Issue by InkDragon** [VoidArkMods/QoLPlus#126]
What the title says, I'd like to request if possible that support be added for [Dino Depot](https://www.curseforge.com/ark-survival-ascended/mods/dino-depot), which is a mod very similar to DinoStorage from ASE.

I understand if this isn't possible, but it would be amazing if you could support other mods that work to preserves buffs from mods like shiny! such as this or Pelayori's Cryo Storage and Super Cryo Storage.

thank you for bring one of the best mods from ASE forward to ASA <3

**Comment by techartguy** [comment]
I second this as well but not sure if it's something that dino depot needs to do or Void has to implement.

**Comment by Rakede** [comment]
Dino depot team here, we''re more than happy to work with mod authors to make dinoballs work with their mods. Since dinoballs aren't cryopods, the mod author would need to pull the values as dinoballs present them.

**Comment by TheBrandonG** [comment]
I can confirm that Pelayori's Cryo Storage does work with the propagator as its how ive been using it

**Comment by InkDragon** [comment]
> I can confirm that Pelayori's Cryo Storage does work with the propagator as its how ive been using it
Super Cryo Storage also works I found, and is what my server is using.

Mostly was wanting dino depot for the admin option to create custom dinos in the ball like dino storage had. Makes for nice gifts from events.

**Comment by Rakede** [comment]
> > I can confirm that Pelayori's Cryo Storage does work with the propagator as its how ive been using it
> > Super Cryo Storage also works I found, and is what my server is using.
> 
> Mostly was wanting dino depot for the admin option to create custom dinos in the ball like dino storage had. Makes for nice gifts from events.

Dino depot does indeed have a custom dino spawn option, link available at the bottom of the curseforge page

**Comment by InkDragon** [comment]
> > > I can confirm that Pelayori's Cryo Storage does work with the propagator as its how ive been using it
> > > Super Cryo Storage also works I found, and is what my server is using.
> > 
> > 
> > Mostly was wanting dino depot for the admin option to create custom dinos in the ball like dino storage had. Makes for nice gifts from events.
> 
> Dino depot does indeed have a custom dino spawn option, link available at the bottom of the curseforge page

yup thats why I mentioned it lol, and why I'd love more mods to support it!
